packages/rx_bloc_cli/mason_templates/bricks/lib_auth/__brick__/lib/lib_auth/services/user_account_service.dart

```diff
@@ -62,7 +62,10 @@ class UserAccountService {
     try {
       final pushToken =
           await _pushSubscriptionRepository.getToken(vapidKey: webVapidKey);
-      if (pushToken != null) {
+      final notificationsSubscribed =
```

Comment:
Is there a reason why we need to keep the `notificationsSubscribed` flag? In my opinion the **pushToken** should always be sent to the backend if available. In cases where the user doesn't want to receive notifications, they can be disabled from the device's settings menu.